package.json

```diff
@@ -87,6 +89,7 @@
   },
   "devDependencies": {
     "autoprefixer": "^10.4.7",
+    "cypress": "^10.6.0",
```

Comment:
Can this be a global package instead of bundling it with app?

Reply:
We need cypress dependency, I don't think without this it will run!